packages/form-builder/src/blocks/fields/amount/inspector/index.js

```diff
@@ -0,0 +1,73 @@
+import {PanelBody} from "@wordpress/components";
+import {__} from "@wordpress/i18n";
+import {InspectorControls} from "@wordpress/block-editor";
+import DeleteButton from "./delete-button";
+import AddButton from "./add-button";
+
+const Inspector = (props) => {
+    return (
+        <InspectorControls>
+            <PanelBody title={__('Donation Levels', 'give')} initialOpen={true}>
+                {props.attributes.levels.length > 0 && (
+                    <ul style={{
+                        listStyleType: 'none',
+                        padding: 0,
+                        display: 'flex',
+                        flexDirection: 'column',
+                        gap: '16px',
+                    }}>
+                        {
+                            props.attributes.levels.map((label, index) => {
+                                return (
+                                    <li key={'level-option-inspector-' + index} style={{
+                                        display: 'flex',
+                                        gap: '16px',
+                                        justifyContent: 'space-between',
+                                        alignItems: 'center',
+                                    }}>
+                                        <div style={{position: 'relative'}}>
+                                            <div
+                                                style={{
+                                                    position: 'absolute',
+                                                    height: '100%',
+                                                    display: 'flex',
+                                                    alignItems: 'center',
+                                                    paddingLeft: '13px',
+                                                    fontSize: '.8em',
+                                                }}>
+                                                <spann style={{marginBottom: '1px'}}>$</spann>
+                                            </div>
```

Comment:
spann! lol

Can we make this currency symbol dynamic?

Reply:
Tag name corrected, see https://github.com/impress-org/givewp-next-gen/pull/49/commits/6e32f27eb950235bec2c2d403c2ccf54334c2c49.

Reply:
I haven't addressed currency in the builder yet (still need to think about currency switcher). Is that something that can wait, for now?

Reply:
Since the currency symbol is dynamic on the front-end, let's make it appear so here too.  We can just use the global currency option for now until we introduce per-form control.  Here's what's happening on the front-end that should be virtually the same here:

<img width="542" alt="Screen Shot 2022-08-08 at 9 13 19 AM" src="https://user-images.githubusercontent.com/10138447/183426348-439a824e-ff79-4ea6-94bc-9d84bcf1bd6d.png">

<img width="840" alt="Screen Shot 2022-08-08 at 9 13 35 AM" src="https://user-images.githubusercontent.com/10138447/183426367-f5caf98b-9e6d-41f5-9d12-1aa2b20e0361.png">

Reply:
@jonwaldstein I localized the currency from the Give settings and implemented `react-currency-input-field`. See https://github.com/impress-org/givewp-next-gen/pull/49/commits/500f5eb22c5bcc75ac7b3ed119604bf74b342ffd.

Reply:
@kjohnson I refactored this a little bit here 72c99d51 just to lean on the `react-currency-input-field` library more and let them handle the currency symbol.  We're really trying to move away from hard-coding the symbols (fyi our currency settings are storing hard-coded symbols) and let js/php generate the locale & symbols based on the currency.

Reply:
Sure that works for me.